sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py

```diff
@@ -208,15 +211,28 @@ def last_detect(self, model_id, variables):
 
     # Last detection
     with open('./sample_data/multivariate_sample_data.json') as f:
-        variables = json.load(f)
-    last_detect_result = sample.last_detect(model_id, variables)
+        variables_data = json.load(f)
+
+    variables = []
+    for item in variables_data['variables']:
+        variables.append(VariableValues(
+            variable = item['variable'],
+            timestamps = item['timestamps'],
+            values = item['values'],
+        ))
+
+    last_inference_body = LastDetectionRequest(
+        variables = variables,
+        top_contributor_count = 10,
+    )
+    last_detect_result = sample.last_detect(model_id, last_inference_body)
 
     assert last_detect_result is not None
 
-    print("Variable States:\t", last_detect_result['variableStates'])
-    print("Variable States length:\t", len(last_detect_result['variableStates']))
-    print("Results:\t", last_detect_result['results'])
-    print("Results length:\t", len(last_detect_result['results']))
+    print("Variable States:\t", last_detect_result.variable_states)
+    print("Variable States length:\t", len(last_detect_result.variable_states))
+    print("Results:\t", last_detect_result.results)
+    print("Results length:\t", len(last_detect_result.results))
```

Comment:
Could you add some test for this new package? Here is the guidance: https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start-for-Test